spec/models/room_spec.rb

```diff
@@ -3,20 +3,32 @@
 RSpec.describe Room, type: :model do
   context "basic tests" do
     let(:room) { FactoryGirl.build(:room) }
+    let(:room2) { FactoryGirl.build_stubbed(:room) }
+
     it "has a valid constructor" do
       expect(room).to be_valid
     end
 
-    it "is not valid without a name" do
+    it "is invalid without a name" do
       room.name = ""
       expect(room).not_to be_valid
       expect(room.errors[:name]).to include "can't be blank"
```

Comment:
Use I18n.t.
This will fail if you change it to Spanish.
See the last changes to test that the translation is there